ui/tests/integration/components/open-api-explorer/swagger-ui-test.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
It looks like the copywrite header was added this backport, so I think we can remove the backport label in this pr https://github.com/hashicorp/vault/pull/23843